### PR TITLE
Remove use of OrderedDict in various places

### PR DESCRIPTION
**Pull request by atombrella**

### CODE REVIEW

File: kombu/connection.py [hunk 1]
```diff
@@ -658,7 +657,7 @@ def _info(self, resolve=True):
 
     def info(self):
         """Get connection info."""
-        return OrderedDict(self._info())
+        return dict(self._info())
```
Comment (auvipy) [inline kombu/connection.py:660]: can you find out some more OrderedDict in the code base please?

**Review by auvipy — CHANGES_REQUESTED** [review]

File: kombu/connection.py (hunk 1 above)
Comment (atombrella) [reply kombu/connection.py:660]: Will do. There are more.

File: kombu/transport/virtual/base.py [hunk 2]
```diff
@@ -160,7 +160,7 @@ class QoS:
     #: current prefetch count value
     prefetch_count = 0
 
-    #: :class:`~collections.OrderedDict` of active messages.
```
Comment (atombrella) [inline kombu/transport/virtual/base.py:163]: Should be reverted.

File: kombu/transport/virtual/base.py [hunk 3]
```diff
@@ -177,6 +177,8 @@ def __init__(self, channel, prefetch_count=0):
         self.channel = channel
         self.prefetch_count = prefetch_count or 0
 
+        # Because we set an attribute on the OrderedDict, which
+        # is not possible with an ordinary dictionary
```
Comment (thedrow) [inline kombu/transport/virtual/base.py:181]: Please rephrase the comment as it isn't using proper English.

**Review by thedrow — CHANGES_REQUESTED** [review]

File: kombu/transport/virtual/base.py (hunk 3 above)
Comment (atombrella) [reply kombu/transport/virtual/base.py:181]: Is this better? I'd prefer not to trigger pipelines just for this.

```
        # Standard Python dictionaries do not support setting attributes
        # on the object, hence the use of OrderedDict
```

Comment (atombrella) [reply kombu/transport/virtual/base.py:181]: @thedrow

Comment (atombrella) [reply kombu/transport/virtual/base.py]: I pushed that change.

**Review by auvipy — APPROVED** [review]

**Pull request merged**